src/components/Ui/IllustratedBanner/IllustratedBanner.tsx

```diff
@@ -1,43 +1,50 @@
 import IllustrationMessageDesign from '@ui5/webcomponents-fiori/dist/types/IllustrationMessageDesign.js';
 import IllustrationMessageType from '@ui5/webcomponents-fiori/dist/types/IllustrationMessageType.js';
-import { FlexBox, IllustratedMessage, Button } from '@ui5/webcomponents-react';
+import { FlexBox, IllustratedMessage, Button, UI5WCSlotsNode } from '@ui5/webcomponents-react';
 import ButtonDesign from '@ui5/webcomponents/dist/types/ButtonDesign.js';
 import '@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js';
+import { ReactElement } from 'react';
 
 type InfoBannerProps = {
   title: string;
-  subtitle: string;
+  subtitle: string | UI5WCSlotsNode;
```

Comment:
I changed the prop to accept both a normal `string` (as before) and a React node (e.g. `<div>` and a `<button>`).